test.disabled/ejabberd_tests/tests/ldap_helper.erl

```diff
@@ -17,7 +17,7 @@
                 ufilter = <<"">>       :: binary(),
                 sfilter = <<"">>       :: binary(),
                 lfilter                :: {any(), any()},
-                deref_aliases = never  :: never | searching | finding | always,
+                deref = never  :: never | searching | finding | always,
```

Comment:
Type spec not updated. Actually, these values should have own, dedicated type defined somewhere.